client/client.go

```diff
@@ -106,22 +106,32 @@ func getProjects(serviceAccountKeyJSON []byte, logger hclog.Logger, filter strin
 	}
 
 	projects := make([]string, 0)
+	inactiveProjects := 0
 	for {
 		output, err := call.Do()
 		if err != nil {
-			return projects, err
+			return nil, err
 		}
 		for _, project := range output.Projects {
 			if project.LifecycleState == "ACTIVE" {
 				projects = append(projects, project.ProjectId)
 			} else {
 				logger.Info("Project state is not active. Project will be ignored", "project_id", project.ProjectId)
+				inactiveProjects++
 			}
 		}
 		if output.NextPageToken == "" {
 			break
 		}
 		call.PageToken(output.NextPageToken)
 	}
+
+	if len(projects) == 0 {
+		if inactiveProjects > 0 {
+			return nil, fmt.Errorf("project listing failed: no active projects")
+		}
```

Comment:
why fail on 1 inactive project,  as we actually set it as info level so it shouldn't fail

Reply:
we have projects, but "all" of them seem to be inactive so we don't have anything left because of this. could easily be a bool too.